_projects/coala_language_server.md

```diff
@@ -42,17 +42,25 @@ a rapidly growing list of editor and language communities:
 We have a state-of-art implementation in
 [coala-vs-code](https://github.com/coala/coala-vs-code) but it is designed
 for VS Code only. We need to design and implement a general mechanism for
-multiple editors/IDEs support.
+multiple editors/IDEs support. We also need clients written to support coala
+langserver for most if not all the supported editors. This is not because of
+the servers inability, but rather the need to configure the front end of the
+editor plugin such as defining how to start the server, communicate with it etc.
 
-The project is divided into two parts:
+The project is divided into three parts:
 
 1. Offer a basic languager server.
 2. Implement a plugin based on the language server.
+3. Optimize the coala entry point and language server.
 
 #### Features
 
 - Support linting for multiple programming languages.
-- Support 'textDocument/publishDiagnostics', 'textDocument/didChange' and 'textDocument/didSave' requests.
+- Support at least the following request types.
```

Comment:
Three of these are already supported?

That should be clarified.

Also we have an issue about workspace requests. Wouldnt we want to support that one also?

Would be appropriate to list some of the request types which are beyond the scope of this project.

Reply:
Some workspace requests such as ```workspace/configuration```, ```workspace/applyEdits``` etc could be very useful for the project. But the way to support ```workspace/workspaceFolders``` is the the one that requires some debate. I however think that support for workspace requests other than the ones included in the updated description file are out of the scope of the project.